siatest/renterhost/renterhost_test.go

```diff
@@ -0,0 +1,100 @@
+package renterhost
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/siatest"
+)
+
+// TestRenterHost executes a number of subtests using the same TestGroup to
+// save time on initialization
+func TestRenterHost(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+
+	// Create a group for the subtests
+	groupParams := siatest.GroupParams{
+		Hosts:   5,
+		Renters: 1,
+		Miners:  1,
+	}
+	tg, err := siatest.NewGroupFromTemplate(groupParams)
+	if err != nil {
+		t.Fatal("Failed to create group: ", err)
+	}
+	defer func() {
+		if err := tg.Close(); err != nil {
+			t.Fatal(err)
+		}
+	}()
+	// Specifiy subtests to run
```

Comment:
probably want to insert some spacing between creating the group and starting the subtests, as I think they are separate thoughts.